reloj_checador_team03/config/webpack/environment.js

```diff
@@ -1,14 +1,3 @@
 const { environment } = require('@rails/webpacker')
 
-
-const webpack = require('webpack')
-environment.plugins.prepend('Provide',
-    new webpack.ProvidePlugin({
-        $: 'jquery/src/jquery',
-        jQuery: 'jquery/src/jquery',
-        Popper: ['popper.js', 'default']
-    })
-)
-
-
-module.exports = environment
```

Comment:
This would still be needed if you depend on bootstrap 4 at least 🤔 

Why are we removing it?